kson-processor/src/main/kotlin/dev/afanasev/kson/processor/KsonTypeAdapterProcessor.kt

```diff
@@ -53,11 +53,12 @@ class KsonTypeAdapterProcessor : KsonProcessor() {
                 .map { it as VariableElement }
                 .map {
                     val key = it.getAnnotation(SerializedName::class.java)?.value ?: it.simpleName
+                    val alternateKeys = it.getAnnotation(SerializedName::class.java)?.alternate?.toList() ?: emptyList()
```

Comment:
Since you are using `it.getAnnotation(SerializedName::class.java)` twice in a row, I think this is a good idea to cache result of this call to variable